ci/build.sh

```diff
@@ -9,10 +9,18 @@ dn=$(dirname $0)
 pkg_install sudo which attr fuse bison \
     libubsan libasan libtsan clang python2 \
     elfutils git gettext-devel libappstream-glib-devel hicolor-icon-theme \
-    dconf-devel fuse-devel \
+    dconf-devel fuse-devel meson dbus-devel \
     /usr/bin/{update-mime-database,update-desktop-database,gtk-update-icon-cache}
 pkg_install_testing ostree-devel ostree
 pkg_install gdk-pixbuf2-modules # needed to make icon validation work
 pkg_install_builddeps flatpak
 
+# malcontent isn’t packaged for our CI distributions yet
+git clone https://gitlab.freedesktop.org/pwithnall/malcontent.git /tmp/malcontent
```

Comment:
We should probably specify a particular tag here rather than "whatever happens to be latest when testing the PR".

Reply:
Good suggestion, specified `0.4.0`.